examples/mem_example/src/main.cpp

```diff
@@ -22,47 +22,117 @@
 #include <stdio.h>
 #include <unistd.h>
 #include <thread>
+#include <iostream>
 
-#include <klepsydra/core/event_emitter_middleware_provider.h>
+#include <klepsydra/high_performance/event_loop_middleware_provider.h>
 #include <klepsydra/mem_vision_ocv/image_multiplexer_vision_ocv_provider.h>
+#include <klepsydra/vision_ocv/file_image_stream_service.h>
 
 #include <klepsydra/mem_core/mem_env.h>
 
 #include <image_processing/image_processor_service.h>
 
 #include "server_user_input.h"
 
-#include "simple_write_service.h"
-
 #include "config.h"
 
+#include "spdlog/spdlog.h"
+
 int main(int argc, char* argv[]) {
     std::string containerName = argc > 1 ? argv[1] : "sample1";
     std::string folderName = argc > 4 ? argv[4] : TEST_DATA;
```

Comment:
This old code needs fixing also. We don't pass any arguments at the moment. So you can make the containerName a constant and have the example only accept 1 argument which is test folder (with TEST_DATA being the default value). You can add a line in the tutorial saying "Execute the example using the command ...." and mention that optionally a test folder can be passed.

Reply:
Done